pkg/apis/networkaddonsoperator/shared/networkaddonsconfig_types.go

```diff
@@ -9,14 +9,15 @@ import (
 // NetworkAddonsConfigSpec defines the desired state of NetworkAddonsConfig
 // +k8s:openapi-gen=true
 type NetworkAddonsConfigSpec struct {
-	Multus                *Multus                `json:"multus,omitempty"`
-	LinuxBridge           *LinuxBridge           `json:"linuxBridge,omitempty"`
-	Ovs                   *Ovs                   `json:"ovs,omitempty"`
-	KubeMacPool           *KubeMacPool           `json:"kubeMacPool,omitempty"`
-	ImagePullPolicy       corev1.PullPolicy      `json:"imagePullPolicy,omitempty"`
-	NMState               *NMState               `json:"nmstate,omitempty"`
-	MacvtapCni            *MacvtapCni            `json:"macvtap,omitempty"`
-	SelfSignConfiguration *SelfSignConfiguration `json:"selfSignConfiguration,omitempty"`
+	Multus                 *Multus                 `json:"multus,omitempty"`
+	LinuxBridge            *LinuxBridge            `json:"linuxBridge,omitempty"`
+	Ovs                    *Ovs                    `json:"ovs,omitempty"`
+	KubeMacPool            *KubeMacPool            `json:"kubeMacPool,omitempty"`
+	ImagePullPolicy        corev1.PullPolicy       `json:"imagePullPolicy,omitempty"`
+	NMState                *NMState                `json:"nmstate,omitempty"`
+	MacvtapCni             *MacvtapCni             `json:"macvtap,omitempty"`
+	SelfSignConfiguration  *SelfSignConfiguration  `json:"selfSignConfiguration,omitempty"`
+	PlacementConfiguration *PlacementConfiguration `json:"placementConfiguration,omitempty"`
```

Comment:
Please add a section to the README describing this new options. Mention the default and show an example how to adjust it.

Reply:
Added Readme section